### ISSUE TITLE
Doesn't honor base path config

### ISSUE THREAD

**Issue by estahn** [banzaicloud/productinfo#92]
Base path has been set to '/' but got following:

```
<!doctype html>
--
  | <html lang="en" xmlns="http://www.w3.org/1999/html">
  | <head>
  | <meta charset="utf-8">
  | <title>Cloud Products</title>
  | <base href="/productinfo/">
  | <meta name="viewport" content="width=device-width, initial-scale=1">

```

**Comment by lpuskas** [comment]
Hi @estahn, could you please provide some context: how did you start the application (docker, docker-compose or you built it yourself)

**Comment by estahn** [comment]
Yes, that would make it easier :)

I used the helm chart with the following config:

```
  - name: flippant-waterbuffalo
    namespace: devops
    chart: banzaicloud-stable/productinfo
    version: 0.4.5
    values:
      - ingress:
          enabled: true
          hosts:
            - productinfo.example.com
          tls:
            - secretName: ***
              hosts:
                - productinfo.example.com
        providers:
          amazon:
            enabled: true
            awsAccessKeyId: ***
            awsSecretAccessKey: ***
```

We're using [helmfile](https://github.com/roboll/helmfile)

**Comment by estahn** [comment]
Same with the curl command in the UI, it doesn't use the actual hostname of the installation. I guess this will need to be passed in, e.g. baseURL.

```
Get the same result with cURL:curl -L -X GET 'https://banzaicloud.com/productinfo/api/v1/providers/amazon/services/compute/regions/ap-southeast-2/products'
```

**Comment by kozmagabor** [comment]
Hi @estahn,
We fixed our image and released 0.4.1 version.
If you want to use with the helm chart, you should set `app.basePath` value like this.
```- name: flippant-waterbuffalo
    namespace: devops
    chart: banzaicloud-stable/productinfo
    version: 0.4.6
    values:
      app.basePath: /yourpath
      ...
```
